.github/actions/setup-environment/action.yaml

```diff
@@ -40,6 +40,9 @@ runs:
     if: runner.os == 'Linux'
     shell: bash
     run: |
+      sudo apt-get update
+      sudo apt-get install pass
```

Comment:
Does this need `--yes` flag to avoid the prompt?

Reply:
No, because `apt` is smart enough to figure out that there's no terminal.  At least https://github.com/mook-as/rd/actions/runs/12308203995/job/34353195839 finishes successfully.